src/couch_views/src/couch_views_indexer.erl

```diff
@@ -483,17 +489,28 @@ report_progress(State, UpdateType) ->
         tx_db := TxDb,
         job := Job1,
         job_data := JobData,
-        last_seq := LastSeq
+        last_seq := LastSeq,
+        db_seq := DBSeq,
+        changes_done := ChangesDone
     } = State,
 
     #{
         <<"db_name">> := DbName,
         <<"db_uuid">> := DbUUID,
         <<"ddoc_id">> := DDocId,
         <<"sig">> := Sig,
-        <<"retries">> := Retries
+        <<"retries">> := Retries,
+        <<"active_tasks_info">> := ActiveTasks
```

Comment:
To avoid hard-coding the field name in all the application, and having a chance someone mistyping it in the future, let's have accessor functions in `fabric2_active_tasks` so that's the only place which knows about `<<"active_task_info">>`. Could have something like `fabric2_active_tasks:update_active_task_info(JobData, ActiveTaskInfo) -> JobData1` and `get_active_task_info(JobData) -> ActiveTaskInfo` for example